tests/err-div-by-zero-imm.data

```diff
@@ -2,5 +2,5 @@
 mov32 r0, 1
 div32 r0, 0
 exit
--- error
-Failed to load code: division by zero at PC 1
+-- result
+0x0
```

Comment:
there seems to be a missing test case for err-mod-by-zero-imm.data which doesn't exist.  Can you create one?

Reply:
Good catch. There was actually a bug here.